tests/api-tests/fields/filter.test.ts

```diff
@@ -252,52 +254,52 @@ multiAdapterRunners().map(({ runner, provider }) =>
                 test(
                   'Less than',
                   withKeystone(({ context }) =>
-                    match(context, { [`${fieldName}_lt`]: storedValues[2][fieldName] }, [0, 1])
+                    match(context, { [fieldName]: { lt: storedValues[2][fieldName] } }, [0, 1])
                   )
                 );
                 test(
                   'Less than or equal',
                   withKeystone(({ context }) =>
-                    match(context, { [`${fieldName}_lte`]: storedValues[2][fieldName] }, [0, 1, 2])
+                    match(context, { [fieldName]: { lte: storedValues[2][fieldName] } }, [0, 1, 2])
                   )
                 );
                 test(
                   'Greater than',
                   withKeystone(({ context }) =>
-                    match(context, { [`${fieldName}_gt`]: storedValues[2][fieldName] }, [3, 4])
+                    match(context, { [fieldName]: { gt: storedValues[2][fieldName] } }, [3, 4])
                   )
                 );
                 test(
                   'Greater than or equal',
                   withKeystone(({ context }) =>
-                    match(context, { [`${fieldName}_gte`]: storedValues[2][fieldName] }, [2, 3, 4])
+                    match(context, { [fieldName]: { gte: storedValues[2][fieldName] } }, [2, 3, 4])
                   )
                 );
               }
               if (mod.supportedFilters(provider).includes('in_empty_null')) {
                 test(
                   'In - Empty List',
-                  withKeystone(({ context }) => match(context, { [`${fieldName}_in`]: [] }, []))
+                  withKeystone(({ context }) => match(context, { [fieldName]: { in: [] } }, []))
                 );
 
                 test(
                   'Not In - Empty List',
                   withKeystone(({ context }) =>
-                    match(context, { [`${fieldName}_not_in`]: [] }, [0, 1, 2, 3, 4, 5, 6])
+                    match(context, { [fieldName]: { notIn: [] } }, [0, 1, 2, 3, 4, 5, 6])
                   )
                 );
 
                 test(
                   'In - null',
                   withKeystone(({ context }) =>
-                    match(context, { [`${fieldName}_in`]: [null] }, [5, 6])
+                    match(context, { [fieldName]: { in: [null] } }, [5, 6])
                   )
                 );
 
                 test(
                   'Not In - null',
                   withKeystone(({ context }) =>
-                    match(context, { [`${fieldName}_not_in`]: [null] }, [0, 1, 2, 3, 4])
+                    match(context, { [fieldName]: { notIn: [null] } }, [0, 1, 2, 3, 4])
```

Comment:
This definitely won't work (prisma doesn't allow null in the array)

Reply:
Yep, this is just to get this closer so that we can evaluate the different against the current APIs and what they support.